src/firefly/js/visualize/ui/DrawLayerUIComponents.jsx

```diff
@@ -50,3 +56,55 @@ export function getMinMaxWidth(maxTitleChars) {
     const maxW = maxTitleChars*.7 > 10 ? Math.min(maxTitleChars*.7, 30) : 10;
     return {minW,maxW};
 }
+
+export function TableColorTitle({color,drawLayerId, plotId, tbl_id}) {
+
+    const table= getTblById(tbl_id);
+    const dl= getDrawLayerById(getDlAry(), drawLayerId);
+    if (!table || !dl) return;
+
+    return (
+        <Stack direction='row' alignItems='center' spacing={1/4} overflow='hidden'>
+            <Button sx={{p:'3px', minHeight:0, zIndex:2}}
```

Comment:
very minor but caught my eye, could possibly be: 
`<Button sx={{p:1/2, minHeight:0, zIndex:2}}`

Reply:
I actually meant 3 pixels. It makes the color button the best size. Here padding is not used for standard spacing.